### ISSUE TITLE
Update npm package to latest version

### ISSUE THREAD

**Issue by KiarashS** [kenwheeler/slick#3462]

**Comment by nwesthoff** [comment]
I agree, 1.8.1 doesn't work with the latest version of JQuery which makes it especially annoying.

**Comment by plato1123** [comment]
I see bower won't grab 1.9.x either, is that because 1.9 is buggy and not ready for release?  (and yes I know I need to replace bower with yarn or npm or...)
"bower slick-carousel#^1.9.0         ENORESTARGET No tag found that was able to satisfy ^1.9.0"

**Comment by pedrosanta** [comment]
Yes, was also surprised when I didn't saw the `1.9.0` published on npm. Latest is the `1.8.1`.

https://www.npmjs.com/package/slick-carousel

I think only @kenwheeler has permissions to publish to npm. @kenwheeler, care to do a quick `npm publish` over there? 😎 Cheers, thanks! 🙌

**Comment by plato1123** [comment]
I can't remember what the issues were but there were some pretty major issues when I tried 1.9.0, better to stay with 1.8.1 for now.

**Comment by pedrosanta** [comment]
Thanks for the heads up @plato1123. 👍

**Comment by ahmadalfy** [comment]
Version 2.0 has been published to npm today
